### DISCUSSION TITLE
No error is reported using the unimported variables console, but the operation is still successful. What is the reason and what needs to be configured

### DISCUSSION THREAD

**Discussion by zhengdechang1** [webpack/webpack#16527] [Q&A]
No error is reported using the unimported variables console, but the operation is still successful. What is the reason and what needs to be configured

![image](https://user-images.githubusercontent.com/118711209/205044165-3b555b5e-9918-4596-8c5d-f83f47ddd98d.png)

**Comment by alexander-akait** [comment]
Can you clarify? They are information messages, maybe you have global `logic`, not sure, please provide example of the problem (github repo?)

**Reply by zhengdechang1** [reply]
repo：https://github.com/corcd/React-Template

All other syntax errors can be prompted, only unintroduced variables can then be used without prompting errors. Only in the browser console can a variable be prompted that does not exist

**Reply by alexander-akait** [reply]
How I can reproducible the problem (commands?)? Because I got `Uncaught ReferenceError: globalStore is not defined` after `yarn install && yarn start`. Also you have `FriendlyErrorsWebpackPlugin` and `WebpackBar`, they can modify your stats output, try to disable them and check it

**Comment by zhengdechang1** [comment]
You can first comment out the globalstore, run the project, enter the home page, comment out one of the import variables, and save it. If there is no syntax error, an error will occur when you open the browser.










      发自我的企业微信         

 
 
 ----------Reply to Message----------
 On Thu, Dec 1, 2022 23:44 PM Alexander ***@***.***&gt; wrote:

  

 
How I can reproducible the problem (commands?)? Because I got Uncaught ReferenceError: globalStore is not defined after yarn install &amp;&amp; yarn start. Also you have FriendlyErrorsWebpackPlugin and WebpackBar, they can modify your stats output, try to disable them and check it

**Reply by alexander-akait** [reply]
Do you expected browsers errors in console? I don't think it is a trivial task, and I think it is out of scope webpack

**Reply by zhengdechang1** [reply]
I want to be able to display undefined variables or syntax errors on the console if I don't introduce them, rather than having them run successfully and see the error in the browser

**Reply by alexander-akait** [reply]
We can't do it, because we don't execute your bundle on Node.js and it is not a browser enviroment

**Reply by zhengdechang1** [reply]
Since the variable is used without introduction, can this be suggested in the console? The previous project was like this, but I don't know which one is controlling this. Could you please try to solve it, or help to provide me with an idea

**Reply by alexander-akait** [reply]
I think you had a linter and it reports to you about unused/undefined variables, look at `eslint`

**Reply by zhengdechang1** [reply]
Ok, thank you
